README.md

```diff
@@ -228,6 +228,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
 
 ## Examples and funny stuff
  * [battlesnake_crystal](https://github.com/nbw/battlesnake_crystal) - Tron snake for [battlesnake](https://www.battlesnake.io/) competition
+ * [blackjack-cr](https://github.com/gdonald/blackjack-cr) - Console Blackjack
```

Comment:
Is this a fully functional game? Should we add it to [Game Development](https://github.com/veelenga/awesome-crystal#game-development) section:

Reply:
It's a fully functional game.  It's nothing to do with game development.  Thanks.